internal/bzlmod/default_gazelle_overrides.bzl

```diff
@@ -22,60 +22,29 @@ DEFAULT_BUILD_FILE_GENERATION_BY_PATH = {
 }
 
 DEFAULT_DIRECTIVES_BY_PATH = {
-    "github.com/census-instrumentation/opencensus-proto": [
-        "gazelle:proto disable",
-    ],
     "github.com/envoyproxy/protoc-gen-validate": [
         "gazelle:build_file_name BUILD.bazel",
-    ],
-    "github.com/cockroachdb/errors": [
-        "gazelle:proto disable",
-    ],
-    "github.com/gogo/googleapis": [
-        "gazelle:proto disable",
-    ],
-    "github.com/gogo/protobuf": [
-        "gazelle:proto disable",
+        "gazelle:proto disable_global",
     ],
     "github.com/google/cel-go": [
         "gazelle:go_naming_convention go_default_library",
-    ],
-    "github.com/google/gnostic": [
-        "gazelle:proto disable",
-    ],
-    "github.com/google/gnostic-models": [
-        "gazelle:proto disable",
+        "gazelle:proto disable_global",
     ],
     "github.com/google/safetext": [
         "gazelle:build_file_name BUILD.bazel",
         "gazelle:build_file_proto_mode disable_global",
-    ],
-    "github.com/googleapis/gax-go/v2": [
-        "gazelle:proto disable",
-    ],
-    "github.com/googleapis/gnostic": [
-        "gazelle:proto disable",
+        "gazelle:proto disable_global",
     ],
     "github.com/pseudomuto/protoc-gen-doc": [
         # The build file in github.com/mwitkow/go-proto-validators has both go_proto and gogo_proto targets, but the checked
         # in go files are generated by gogo proto. Resolving to the gogo proto target preserves the behavior of Go modules.
         "gazelle:resolve go github.com/mwitkow/go-proto-validators @com_github_mwitkow_go_proto_validators//:validators_gogo",
-    ],
-    "google.golang.org/grpc": [
-        "gazelle:proto disable",
-    ],
-    "google.golang.org/protobuf": [
-        "gazelle:proto disable",
-    ],
-    "k8s.io/api": [
-        "gazelle:proto disable",
-    ],
-    "k8s.io/apiextensions-apiserver": [
-        "gazelle:proto disable",
+        "gazelle:proto disable_global",
     ],
     "k8s.io/apimachinery": [
         "gazelle:go_generate_proto false",
         "gazelle:proto_import_prefix k8s.io/apimachinery",
+        "gazelle:proto disable_global",
```

Comment:
Let's don't disable proto here. People will need the proto targets from this module, otherwise they will have to download it again with `http_archive`.